hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -427,7 +427,7 @@ public final class ScmConfigKeys {
   public static final String OZONE_SCM_PIPELINE_SCRUB_INTERVAL =
       "ozone.scm.pipeline.scrub.interval";
   public static final String OZONE_SCM_PIPELINE_SCRUB_INTERVAL_DEFAULT =
-      "5m";
+      "60s";
```

Comment:
This can have unintended consequence. We do not know how long a pipeline stays in `ALLOCATED` state.

Reply:
We have a separate property for handling pipelines in ALLOCATED state.
Since the scrubber thread is handling both ALLOCATED and CLOSED pipelines, it should have a lower interval.